apps/aecontract/src/aect_contracts.erl

```diff
@@ -205,15 +205,13 @@ serialize(#contract{owner_id     = OwnerId,
 serialize_for_client(#contract{id           = Id,
                                owner_id     = OwnerId,
                                ct_version   = CTVersion,
-                               log          = Log,
                                active       = Active,
                                referrer_ids = ReferrerIds,
                                deposit      = Deposit}) ->
     #{ <<"id">>           => aehttp_api_encoder:encode(id_hash, Id)
      , <<"owner_id">>     => aehttp_api_encoder:encode(id_hash, OwnerId)
      , <<"vm_version">>   => maps:get(vm, CTVersion)
      , <<"abi_version">>  => maps:get(abi, CTVersion)
-     , <<"log">>          => Log
```

Comment:
I see this function used also in HTTP API, that declares this field as [required](https://github.com/aeternity/aeternity/blob/f5c46b80561697c2c41d03f9cc2dc7400b153e9e/config/swagger.yaml#L3102). Removing this from HTTP API seems backward incompatibility cc @noandrea - even if little - and we may postpone it ([tracked](https://www.pivotaltracker.com/story/show/164539296) anyway). Though anyway if we change it in code we need to update swagger schema.

May I suggest in this PR to keep user HTTP API unchanged, and just not expose this fild in websocket? (I understand not cleanest in code though least impact on API.)

Reply:
Ok, I will do that!